packages/@groupby/elements-search-driver-plugin/test/unit/common/search-driver-plugin.test.ts

```diff
@@ -85,45 +85,56 @@ describe('SearchDriverPlugin', () => {
   });
 
   describe('fetchSearchData()', () => {
+    let area;
+    let collection;
+    let config;
     let results;
     let group;
     let sendSearchApiRequest;
+    let query;
 
     beforeEach(() => {
-      results = { a: 'a' };
+      area = 'area';
+      collection = 'collection';
```

Comment:
Since these are strings (and therefore primitives), you can turn the variables into `const`s and initialize them outside of the `beforeEach`.